proposals/006-spinapp-invocation-limits/README.md

```diff
@@ -0,0 +1,70 @@
+# SKIP 006 - SpinApp invocation limits
+
+Summary: Add support for configuring per-invocation resource limits in `SpinApp` CRD
+
+Owner: Kate Goldenring <kate.goldenring@fermyon.com>
+
+Impacted Projects:
+
+- [x] spin-operator
+- [ ] `spin kube` plugin
+- [ ] runtime-class-manager
+- [ ] containerd-shim-spin
+- [ ] Governance
+- [ ] Creates a new project
+
+Created: 05-29-2025
+
+Updated: 05-29-2025
+
+## Background
+
+Spin recently [added support for limiting instance memory](https://github.com/spinframework/spin/pull/3135) and has had an outstanding request for supporting [request timeouts](https://github.com/spinframework/spin/issues/30). The `SpinApp` CRD should support configuring per invocation limits.
+
+## Proposal
+
+Add an `invocationLimits` section to the `SpinApp` CRD with a mapping of limits. The limits can be of two types: common and executor namespaced.The project will define common keys that executors *should* support, starting with `memory`. Executors can define additional limits that they support as namespaced keys; for example, the key for a `bar` limit specific to the Spintainer executor may be `spintainer.spinframework.dev/bar`.
+
+Tje Spin Operator will validate common limits. For `memory`, it will assert that the requested value does not exceed that of the `resources.limits.memory` section. Executors are responsible for documenting and validating their limits. One approach is to provide a validating webhook for the `SpinApp` resource to avoid unaccepted values being passed.
```

Comment:
```suggestion
The Spin Operator will validate common limits. For `memory`, it will assert that the requested value does not exceed that of the `resources.limits.memory` section. Executors are responsible for documenting and validating their limits. One approach is to provide a validating webhook for the `SpinApp` resource to avoid unaccepted values being passed.
```